src/components/two-up-display.tsx

```diff
@@ -58,6 +58,7 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
         <div className="content scrollable" data-test="right-content">
           {this.props.rightPanel}
         </div>
+        <div className="footer" data-test="right-footer"/>
```

Comment:
Looking back on this, I'm not certain that these footer elements quite make sense like this, because `rightPanel` can't easily put elements in them. Also I'm not sure that they should show for instructions.

Reply:
They are in all panels in the UI spec, but I don't like them.  They take up space and provide no functionality.  They seem to only exist for aesthetics.

Reply:
Ok, yeah, I think they're supposed to eventually have UI in them, but until then they are taking up space. So I think I will remove them in my branch until we've added functionality to them.